<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QComboBox" name="proteinList">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>124</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #333333;
	border-radius: 3px;
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
	padding: 1px 23px 1px 3px;
	min-width: 6em;
	color: #ffffff;</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="riceList">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>124</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #333333;
	border-radius: 3px;
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
	padding: 1px 23px 1px 3px;
	min-width: 6em;
	color: #ffffff;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="beansList">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>124</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #333333;
	border-radius: 3px;
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
	padding: 1px 23px 1px 3px;
	min-width: 6em;
	color: #ffffff;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chipsList">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #333333;
	border-radius: 3px;
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
	padding: 1px 23px 1px 3px;
	min-width: 6em;
	color: #ffffff;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="topping1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>124</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #333333;
	border-radius: 3px;
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
	padding: 1px 23px 1px 3px;
	min-width: 6em;
	color: #ffffff;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cheeseList">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>124</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #333333;
	border-radius: 3px;
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
	padding: 1px 23px 1px 3px;
	min-width: 6em;
	color: #ffffff;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 13pt &quot;Futura&quot;;
color: rgb(83, 207, 255);
qradialgradient: (spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));</string>
    </property>
    <property name="text">
     <string>Select a protein.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 13pt &quot;Futura&quot;;
color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 255), stop:0.19397 rgba(0, 0, 0, 255), stop:0.202312 rgba(122, 97, 0, 255), stop:0.495514 rgba(76, 58, 0, 255), stop:0.504819 rgba(255, 255, 255, 255), stop:0.79 rgba(255, 255, 255, 255), stop:1 rgba(255, 158, 158, 255));</string>
    </property>
    <property name="text">
     <string>Select a rice.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 205, 39);
font: 900 13pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Select beans.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(241, 127, 255);
font: 700 13pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Select a topping.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(161, 148, 255);
font: 700 13pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Select a cheese.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 13pt &quot;Futura&quot;;
color: rgb(255, 111, 123);</string>
    </property>
    <property name="text">
     <string>Chips?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 24pt &quot;Optima&quot;;
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.001 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));</string>
    </property>
    <property name="text">
     <string>Calculate!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rendelogo.qrc">:/images/UCLA_logo_PNG1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>441</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>CHICKEN Pie Height</family>
      <pointsize>72</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(38, 56, 255);
font: 72pt &quot;CHICKEN Pie Height&quot;;
</string>
    </property>
    <property name="text">
     <string>Rendezvous</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rendelogo.qrc">:/images/ac.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;CHICKEN Pie Height&quot;;
color: rgb(116, 208, 31);</string>
    </property>
    <property name="text">
     <string>FoodPrint Calculator: Just input your rende order!</string>
    </property>
   </widget>
   <widget class="QLabel" name="newOne">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 50pt &quot;Bodoni 72&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;CHICKEN Pie Height&quot;;</string>
    </property>
    <property name="text">
     <string>Your order's climate score is...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="indent">
     <number>-3</number>
    </property>
   </widget>
   <widget class="QLabel" name="YEAH">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>12777215</width>
      <height>12777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;CHICKEN Pie Height&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 24pt &quot;Optima&quot;;
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.001 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
background-color: &quot;white&quot;;</string>
    </property>
    <property name="text">
     <string>Show me the data</string>
    </property>
   </widget>
   <zorder>label_11</zorder>
   <zorder>proteinList</zorder>
   <zorder>riceList</zorder>
   <zorder>beansList</zorder>
   <zorder>chipsList</zorder>
   <zorder>topping1</zorder>
   <zorder>cheeseList</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_10</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>newOne</zorder>
   <zorder>YEAH</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="rendelogo.qrc"/>
 </resources>
 <connections/>
</ui>
